index.html

```diff
@@ -3852,9 +3852,14 @@ <h2 id="AudioParam">
             <p>
               Cancels all scheduled parameter changes with times greater than
               or equal to <code>cancelTime</code>. Active
-              <code><a>setTargetAtTime</a></code> automations (those with
-              <code>cancelTime</code> less than the supplied time value) will
-              also be cancelled. Any hold values scheduled by <a href=
+              <code><a>linearRampToValueAtTime</a></code>,
+              <code><a>exponentialRampToValueAtTime</a></code>,
+              <code><a>setTargetAtTime</a></code> and
+              <code><a>setValueCurveAtTime</a></code> automations (those with
+              <code>cancelTime</code> less than the automation's end time) will
```

Comment:
Yes, this sounds better. I agree it is unnecessary to list every automation method. How about your suggestion with the addition "...scheduled to occur or end after `cancelTime`, even if..."

Because to me, currently running automations do not occur completely after `cancelTime`, but also before (partially).

Sorry for the nitpicking ;-)